src/GraphQL/helpers/objectsQueries.js

```diff
@@ -3,6 +3,11 @@ import { offsetToCursor, cursorToOffset } from 'graphql-relay';
 import rest from '../../rest';
 import { transformQueryInputToParse } from '../transformers/query';
 
+const hasCustomField = (fields, keys) =>
+  keys
+    ? !!keys.split(',').find(keyName => !fields[keyName.split('.')[0]])
+    : true;
```

Comment:
Is this expected, no `key` means there _are_ custom fields?

Reply:
Bad naming yes, i fixed it !